ENTERPRISE-MIB

```diff
@@ -0,0 +1,151 @@
+CLEARWATER-ALARMS-MIB DEFINITIONS ::= BEGIN
+
+-- /**********************************************************/
+-- /* Note: Some strict MIB compilers require 'ObjectID'     */
+-- /* to be replaced with 'OBJECT IDENTIFIER'                */
+-- /**********************************************************/
+
+IMPORTS
+        MODULE-IDENTITY, OBJECT-TYPE, Integer32, enterprises,
+        NOTIFICATION-TYPE
+                                            FROM SNMPv2-SMI
+        TEXTUAL-CONVENTION, TruthValue
```

Comment:
I think you need to import the DisplayString textual convention as well. (I think it's worth you running this MIB through smilint before I do further live testing).